src/components/story-block.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import Media from 'react-bootstrap/Media'
+import Container from 'react-bootstrap/Container'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
+import Img from 'gatsby-image'
+
+const StoryBlock = ({
+  imageSource,
+  altText,
+  description,
+  imageRight = false,
+}) => (
+  <section>
+    <Container>
+      <Row>
+        <Col sm={{ span: '6', order: `${imageRight ? 'last' : 'first'} ` }}>
+          <Img fluid={imageSource} alt={altText} />
```

Comment:
using the new gatsby image component, called 'Img'

Reply:
Helps control the image?

Reply:
Img does all the fancy optimization stuff: 
https://www.gatsbyjs.org/packages/gatsby-image/